src/components/HomeView/HomeView.styled.jsx

```diff
@@ -1,8 +1,9 @@
 import styled from 'styled-components';
 
 const HomeViewWrapper = styled.div`
-  background-color: #ffffff;
   align-items: center;
+  background-color: ${(props) => (props.theme === 'light' ? '#ffffff' : '#303030')};
```

Comment:
Way easier to do this if you handle a theme using styled-components and color variables :D